README.md

```diff
@@ -45,9 +45,9 @@ $ yarn add --dev eslint-plugin-testing-library
 
 **Note:** If you installed ESLint globally (using the `-g` flag) then you must also install `eslint-plugin-testing-library` globally.
 
-## Migrating to v4
+## Migrating
 
-You can find [here a detailed guide for migrating `eslint-plugin-testing-library` to v4](docs/migrating-to-v4-guide.md).
+You can find detailed guides for migrating `eslint-plugin-testing-library` in the [migration guide docs](docs/migration-guides).
```

Comment:
I think we can add here a list of links, only with the v4 migration guide for now.